docs/guide/index.md

```diff
@@ -231,6 +232,17 @@ You can pass the `preload_stylesheet` variable to the `vite-tag` snippet to enab
 {% render 'vite-tag' with 'theme.scss', preload_stylesheet: true %}
 ```
 
+### Defer Loading Stylesheets
+
+You can pass the `preload_stylesheet: 'defer'` variable to the `vite-tag` snippet to defer loading stylesheets. For example, consider defer loading non-critical stylesheets.
+[Learn more](https://web.dev/articles/defer-non-critical-css).
+
+You can also pass the `fetchpriority` variable to the `vite-tag` snippet to indicate priority of deferred stylesheets. [Learn more](https://web.dev/articles/fetch-priority)
```

Comment:
@tuanpham-barrel Would it be helpful to clarify that this option only applies when passing in the 'defer' value?

Reply:
@montalvomiguelo I've just updated the doc